src/ui/template/restaurant.html

```diff
@@ -202,11 +262,11 @@ <h2>The priority order for each employee in each role</h2>
                     size=8
                     pattern="\d\d?:\d\d [AaPp][mM]"
                     title="End time"
-                    requred/>
+                    required/>
             <br/>
             <textarea name="note" rows="4" cols="50" placeholder="Notes for scheduler if needed"></textarea>
             <br/>
-            <input type="submit" value="add availability"/>
+            <input type="submit" id="addavail" value="Add Availability"/>
```

Comment:
It's ok to do corrections, but let's have their own pull request for cleanups like this.

Reply:
Ok got it.